components/mdc/Switch/Switch.svelte

```diff
@@ -0,0 +1,73 @@
+<!--  https://github.com/material-components/material-components-web/tree/master/packages/mdc-switch -->
+<script>
+import { MDCSwitch } from '@material/switch'
+import { createEventDispatcher, onMount } from 'svelte'
+
+/**
+ * displayed right of the switch
+ */
+export let label = 'off/on'
+/**
+ * disables the switch
+ */
+export let disabled = false
+/**
+ * used to set the switch initial state
+ */
+export let selected = false
+
+let element = {}
+let switchControl = {}
+
+onMount(() => {
+  switchControl = new MDCSwitch(element)
+})
+
+const dispatch = createEventDispatcher()
+
+// Used setTimeout because switchControl.selected needs to be updated before dispatching the event
+function onClick() {
+  setTimeout(() => dispatch(switchControl.selected ? 'selected' : 'deselected'))
+}
+</script>
+
+<!--
+  @component
+  Used to toggle an item on tablet and mobile devices or immediately activate/deactive something
+  see https://silinternational.github.io/ui-components/?path=/docs/atoms-switch--primary for usage
+-->
+
+<button
+  on:click={onClick}
+  bind:this={element}
+  id="basic-switch"
```

Comment:
You may want to use our `generateRandomID()` method to come with an id for this, to avoid `id` conflicts if someone uses more than one of these Switch components on a single page.